package.json

```diff
@@ -33,16 +33,17 @@
         "publish": "vsce publish"
     },
     "dependencies": {
-        "testyts": "0.7.x",
+        "jsonc-parser": "^2.2.1",
```

Comment:
Typescript comes with a "readConfigFile" method, I'd rather use it than add an external dependency.